homeassistant/components/lifx/select.py

```diff
@@ -30,11 +46,16 @@ async def async_setup_entry(
         async_add_entities(
             [
                 LIFXInfraredBrightnessSelectEntity(
-                    coordinator, description=INFRARED_BRIGHTNESS_ENTITY
+                    coordinator=coordinator, description=INFRARED_BRIGHTNESS_ENTITY
                 )
             ]
         )
 
+    if lifx_features(coordinator.device)["multizone"] is True:
+        async_add_entities(
```

Comment:
We could collect the entities in a list from both cases and add them in a single call to `async_add_entities`.

Reply:
Yes, thats much better as it avoids creating multiple tasks 

Since I already had this open I took care of these https://github.com/home-assistant/core/pull/80828